react/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
Ignore file should be moved to the root folder so it enacts on the entire repo. Additionally, it should include some files/folders related to rails such as the `tmp` and `log` folders. Find more https://www.toptal.com/developers/gitignore/api/rails or across the web or even rails repo.

Reply:
Done.